client.go

```diff
@@ -397,7 +397,7 @@ func (c *Client) ImportField(field *Field, iterator RecordIterator, options ...I
 			importRecordsFunction(c.importValues)(&importOptions)
 		} else {
 			// Check whether roaring imports is available
-			if importOptions.wantRoaring {
+			if importOptions.wantRoaring != nil && *importOptions.wantRoaring == true {
```

Comment:
we could possibly drop this, and the section around 1500, by treating nil as true. so `if importOptions.wantRoaring == nil || *importOptions.wantRoaring == true`, maybe?

Reply:
I thought about moving the default check out of `withDefaults` but that forces us to remember the default, which may be problematic if we check against `importOptions.wantRoaring` elsewhere.

Reply:
Hmm. That's true. So in this case, the != nil isn't intended as part of the decision logic, it's a sanity-check to prevent panics if something went wrong elsewhere?

Reply:
Sorry, I've merged the PR before I saw your comment. Yeah, I've just added it for piece of mind.